evap/rewards/views.py

```diff
@@ -158,6 +159,37 @@ def reward_point_redemption_event_export(request, event_id):
     return response
 
 
+@manager_required
+def reward_points_export(request):
+    filename = _("RewardPoints") + f"-{get_language()}.csv"
+    response = AttachmentResponse(filename, content_type="text/csv")
+
+    writer = csv.writer(response, delimiter=";", lineterminator="\n")
+    writer.writerow(
+        [
+            _("Email address"),
+            _("Number of points"),
```

Comment:
Removing the trailing comma in the list here (and also in the `for profile` loop below) will make our formatter put this in one line, which I would prefer :)

Reply:
Done!